scripts/list_apis.py

```diff
@@ -0,0 +1,195 @@
+import argparse
+import inspect
+import importlib
+import sys
+
+from types import ModuleType
+from collections import namedtuple
+from typing import List, Any, Set
+
+
+CrawlError = namedtuple("CrawlError", ("reason", "path"))
+CrawlResult = namedtuple("CrawlResult", ("public", "private", "errors", "module"))
+
+
+def err_print(*args: Any) -> None:
```

Comment:
There are a lot of error printing happening here. Do we want to actually keep that? Or should we silence these? Or have a list of expected errors?